Dockerfile.f30-mini

```diff
@@ -0,0 +1,5 @@
+FROM registry.fedoraproject.org/fedora-minimal:30 
```

Comment:
What's the need for using a fedora base-image?

Reply:
we want to deploy the application on an OpenShift based on Fedora, therefore we want to base all the container images on Fedora. If this is too far away from what you want I can keep the changes in my fork...

Reply:
I think it's OK. If it's useful anyone can use it, if not they can ignore it.